experimental/attention_call.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import jax
+import jax.numpy as jnp
+import attention
+from jax._src.interpreters import mlir as jax_mlir
+from jax._src.lib.mlir import ir
+from jax.experimental import export
+
+
+def export_iree_attention(query, key, value, scale):
+  inputs = (query_in, key_in, value_in, scale_in)
+  input_shapes = [
+      jax.ShapeDtypeStruct(input.shape, input.dtype) for input in inputs
+  ]
+  att = export.export(
```

Comment:
Should this be some generic IREE export somewhere? E.g., have list of "allowed for IREE export" custom calls listed somwehere. Then one always does iree.jax.export instead of jax.experimental.export.